### PR TITLE
fix ekco reboot service start

### PR DESCRIPTION
**Pull request by laverya**
#### What this PR does / why we need it:

If k8s is unhealthy upon starting, the ekco start service can get 'stuck' starting. The solution is to restart the service, and this changes the script to do so when installing ekco.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by camilamacedo86 — APPROVED** [review]

**Review by rrpolanco — APPROVED** [review]
appreciate the additional logging

File: addons/ekco/template/base/install.sh [hunk 1]
```diff
@@ -154,6 +154,8 @@ function ekco_maybe_remove_rook_priority_class_label() {
 function ekco_install_reboot_service() {
     local src="$1"
 
+    echo "Installing ekco reboot service"
```
[the remark this answers is not in the document]
Comment (laverya) [reply addons/ekco/template/base/install.sh:157]: maybe, but IMO that should be its own change (the rest of this file uses 'echo')

**Review by camilamacedo86 — APPROVED** [review]

File: addons/ekco/0.26.5/install.sh [hunk 2]
```diff
@@ -167,14 +169,23 @@ function ekco_install_reboot_service() {
 
     systemctl daemon-reload
     systemctl enable ekco-reboot.service
-    systemctl start ekco-reboot.service
+    if ! timeout 30s systemctl start ekco-reboot.service; then
+        echo "Failed to start ekco-reboot.service within 30s, restarting it"
+        systemctl restart ekco-reboot.service
+    fi
+
+    echo "ekco reboot service installed"
```
Comment (camilamacedo86) [inline addons/ekco/0.26.5/install.sh:177]:
```suggestion
    logSuccess "ekco reboot service installed"
```

File: addons/ekco/0.26.5/install.sh [hunk 3]
```diff
@@ -167,14 +169,23 @@ function ekco_install_reboot_service() {
 
     systemctl daemon-reload
     systemctl enable ekco-reboot.service
-    systemctl start ekco-reboot.service
+    if ! timeout 30s systemctl start ekco-reboot.service; then
+        echo "Failed to start ekco-reboot.service within 30s, restarting it"
```
Comment (camilamacedo86) [inline addons/ekco/0.26.5/install.sh:173]:
```suggestion
        logFail "Failed to start ekco-reboot.service within 30s, restarting it"
```

File: addons/ekco/0.26.5/install.sh [hunk 4]
```diff
@@ -167,14 +169,23 @@ function ekco_install_reboot_service() {
...
 
 function ekco_install_purge_node_command() {
     local src="$1"
 
+    echo "Installing ekco purge node command"
+
     cp "$src/ekco-purge-node.sh" /usr/local/bin/ekco-purge-node
     chmod u+x /usr/local/bin/ekco-purge-node
+
+    echo "ekco purge node command installed"
```
Comment (camilamacedo86) [inline addons/ekco/0.26.5/install.sh:188]:
```suggestion
    logSuccess "ekco purge node command installed"
```

File: addons/ekco/0.26.5/install.sh [hunk 5]
```diff
@@ -458,9 +469,13 @@ function ekco_load_images() {
...
 
+    echo "Loading ekco image overrides"
+
     if [ -n "$DOCKER_VERSION" ]; then
         find "$DIR/image-overrides" -type f | xargs -I {} bash -c "docker load < {}"
     else
         find "$DIR/image-overrides" -type f | xargs -I {} bash -c "cat {} | ctr -a $(${K8S_DISTRO}_get_containerd_sock) -n=k8s.io images import -"
     fi
+
+    echo "ekco image overrides loaded"
```
Comment (camilamacedo86) [inline addons/ekco/0.26.5/install.sh:480]:
```suggestion
    logSuccess "ekco image overrides loaded"
```

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Review by rrpolanco — APPROVED** [review]

**Pull request merged**